util/get-toolchain.py

```
@@ -37,7 +37,11 @@
 
 def get_available_toolchain_info(version, kind):
     assert kind in ASSET_PREFIXES
-
+    
+    import platform
+    # Check if we're on x86_64
+    arch = "x86_64" if platform.machine() == 'x86_64' else "aarch64"
```

I think it would be better to handle explicitely a list of supported platforms and fail if it's not in the listed, rather than defaulting to `aarch64`.